libs/remix-ui/static-analyser/src/lib/Checkbox/StaticAnalyserCheckedBox.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react' //eslint-disable-line
+
+interface StaticAnalyserCheckBoxProps {
```

Comment:
We can extract this checkbox as a custom checkbox reusable component.

Reply:
checkbox is now a reusable component